app/controllers/question_controller.rb

```diff
@@ -9,10 +9,10 @@ class QuestionsController < ApplicationController
 	respond_to :html, :json
 
 	def index
-		@questions = Question.all
-	end	
+		@questions = Question.order("id DESC")
```

Comment:
Woot! I also added it to answer.rb. I figure since we'll be filtering by question id in the views, this will work as expected right?
```ruby
class Answer < ActiveRecord::Base
  belongs_to :user
  belongs_to :question
  has_many :comments
  default_scope { order(created_at: :desc) }
end
```